.github/workflows/ci.yml

```diff
@@ -281,6 +283,8 @@ jobs:
       DEPS_BUILD_DIR: ${{ github.workspace }}/nvim-deps
       CACHE_NVIM_DEPS_DIR: ${{ github.workspace }}/nvim-deps
       DEPS_PREFIX: ${{ github.workspace }}/nvim-deps/usr
+      # TEST_FILE: test/functional/core/startup_spec.lua
+      # TEST_FILTER: foo
```

Comment:
Leaving this here and documented in CONTRIBUTING.md to help others.

Reply:
Just put one set of environment variables at the top level instead?

Reply:
That didn't seem to work when I tried it...